docs/interview/common-questions/README.md

```diff
@@ -24,9 +24,72 @@ Roadmap Module: [Welcome to the DevOps World](../../foundations/module-01)
 
 Roadmap Module: [Basics - DevOps Core](../../foundations/module-02)
 
-**Work in progress**
+- What are Python’s key features that make it suitable for Cloud and DevOps?
```

Comment:
@Mohamed-abdalazez Rebase to main, that part is already added (always checkout from the upstream main branch not from your main branch).